<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerPinnacleDVFReaderOptionsWidget</class>
 <widget class="qSlicerWidget" name="qSlicerPinnacleDVFReaderOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume Options</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Grid Origin (XYZ):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="gridOriginX">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="gridOriginY">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="gridOriginZ">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
